src/pages/SupplierRequisitionPage.js

```diff
@@ -63,10 +75,14 @@ export class SupplierRequisitionPage extends React.Component {
 
   onCreateAutomaticOrder = () => {
```

Comment:
if we use `Use Suggested Quantities` button, logic in the method is not needed

Reply:
done 4dab3938830fa467815b1fc17fbe120d8781c7a7